techdocs/script/nodes.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+
+# Does not check the file on disk, but uses the path to determine the type
+def looks_fileish(fspath):
+    return "." in os.path.basename(fspath)
+
+def looks_dirish(fspath):
+    return not looks_fileish(fspath)
```

Comment:
That looks suspicioish ;D

why not using `os.path.isfile()` / `os.path.isdir()` ?

`looks_dirish` could have a false positive with hidden directories like `.git`

Some problems with tests? I see that you have such methods after all:
https://github.com/PiwikPRO/actions/blob/3fc4547188eb8b25bd5efc3cc257424c20eada8e/techdocs/script/filesystem.py#L9-L13